tests/unit/components/buttons/HdRadioButton.spec.js

```diff
@@ -24,24 +23,26 @@ describe('HdRadioButton', () => {
         mobileIcon,
       },
     });
-
-    wrapper.setMethods({ select: mockedSelect });
   });
 
   test('the component is rendered', () => {
     expect(wrapper.html()).toMatchSnapshot();
   });
 
+  test('test the background-image of the radio icon', () => {
+    expect(wrapper.find('.radioButton__icon').attributes().style).toBe('background-image: url(desktop-icon-url);');
+  });
+
   test('on change, the proper method is invoked', () => {
     const input = wrapper.find('input');
-
     expect(input.attributes().name).toBe(name);
     expect(input.attributes().value).toBe(value);
-
     expect(wrapper.find('label').text()).toBe(label);
 
+    // Check events emitted before and after change
+    expect(wrapper.emitted('select')).toBeFalsy();
     input.trigger('change');
-
-    expect(mockedSelect).toHaveBeenCalled();
+    expect(wrapper.emitted('select')).toBeTruthy();
```

Comment:
```suggestion
```

You can delete this line as the new assertion already covers it.

Reply:
@ilyasmez but then the test wouldn't be 100% sure that we only emit the "select" event after a "change". It can look like it could have been fired after the component was created.
With both, we make sure this is triggered only after the change event.

Reply:
Ah my bad, sorry!! I overlooked this last line: 
```js
expect(wrapper.emitted('select')[0][0]).toMatchObject({ value, name });
```
My comment was for @viniciuskneves because I wasn't sure about the reason of removing that line (I didn't see the line above 🙈 ).

Reply:
Done 👍